src/cluster.c

```diff
@@ -172,24 +175,25 @@ typedef struct {
 /* Assign index to each aux field */
 typedef enum {
     af_start,
-    af_shard_id = af_start,
-    af_count,
+    af_shard_id = af_start+1,
+    af_count = 2,
```

Comment:
```suggestion
    af_shard_id = af_start,
    af_human_nodename,
    af_count,
```
I think the intention is we would add these incrementally. I don't think this actually matters, but for future aux fields I assume we'll want this PR to set a good precedent for how to add more.

Reply:
@madolson I believe we don't need the af_start as well. as we are only using it for for loop initialization and no where as else. If It is okay, I would like to remove it.

Reply:
Sounds OK to me.

Reply:
Great, I have updated all the changes related to enum !